apps/web/components/twitter-editor.tsx

```diff
@@ -104,8 +104,8 @@ export default function TwitterEditor() {
   const {
     previewHeightPixels,
     previewWidthPixels,
-    width: previewWidth,
-    height: previewHeight,
+    width: Width,
+    height: Height,
```

Comment:
not like this, it should look like:
see `apps/web/components/gradient-editor.tsx`